modules/expo-bluesky-oauth-client/index.ts

```diff
@@ -1,6 +1,4 @@
-export * from './src/crypto-subtle'
-export * from './src/jose-key'
-export * from './src/rn-crypto-key'
-export * from './src/rn-oauth-client-factory'
-export * from './src/rn-oauth-database'
-export * from './src/util.web'
+export * from './src/oauth-client-react-native'
+export * from './src/react-native-crypto-implementation'
+export * from './src/react-native-key'
+export * from './src/react-native-store-with-key'
```

Comment:
Note, this is probably a good place to expose the "universal" module by having an `index.web.ts` that exposes the same interfaces/classes (but from `@atproto/oauth-client-browser`) ?